src/test/java/org/openhab/tools/analysis/checkstyle/test/BuildPropertiesExternalLibrariesCheckTest.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.tools.analysis.checkstyle.test;
+
+import java.io.File;
+
+import org.junit.Test;
+import org.openhab.tools.analysis.checkstyle.BuildPropertiesExternalLibrariesCheck;
+import org.openhab.tools.analysis.checkstyle.api.AbstractStaticCheckTest;
+
+import com.puppycrawl.tools.checkstyle.DefaultConfiguration;
+import com.puppycrawl.tools.checkstyle.TreeWalker;
+import com.puppycrawl.tools.checkstyle.api.Configuration;
+import com.puppycrawl.tools.checkstyle.utils.CommonUtils;
+
+/**
+ * Test for the {@link BuildPropertiesExternalLibrariesCheck}
+ *
+ * @author Velin Yordanov - initial contribution
+ *
+ */
+public class BuildPropertiesExternalLibrariesCheckTest extends AbstractStaticCheckTest {
+    private static DefaultConfiguration configuration = createCheckConfig(BuildPropertiesExternalLibrariesCheck.class);
+    private static String TEST_FILE_NAME = "build.properties";
+    private static final String MAIN_DIRECTORY = "buildPropertiesExternalLibrariesCheck";
+
+    @Override
+    protected DefaultConfiguration createCheckerConfig(Configuration config) {
+        DefaultConfiguration configParent = createCheckConfig(TreeWalker.class);
+        configParent.addChild(config);
+        return configParent;
+    }
+
+    @Test
+    public void shouldNotLogWhenBundleIsValidFolder() throws Exception {
+        final String VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES = MAIN_DIRECTORY + File.separator + "validBundle";
+        String[] warningMessages = CommonUtils.EMPTY_STRING_ARRAY;
+        String filePath = getPath(VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES + File.separator + TEST_FILE_NAME);
+        verify(configuration, filePath, warningMessages);
+    }
+
+    @Test
+    public void shouldNotLogWhenBundleIsValidEntries() throws Exception {
+        final String VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES = MAIN_DIRECTORY + File.separator
+                + "validBundleWithJarFilesInBuildProperties";
+        String[] warningMessages = CommonUtils.EMPTY_STRING_ARRAY;
+        String filePath = getPath(VALID_BUNDLE_WITH_JAR_FILES_IN_BUILD_PROPERTIES + File.separator + TEST_FILE_NAME);
```

Comment:
I don't know if there is a common pattern across all usages of the `verify` method but every caller of the method has to call `getPath` first, we could consider to make that part of the `verify` method.